docs/content/docs/start/tree-start.md

```diff
@@ -0,0 +1,239 @@
+---
+title: SharedTree Quick Start
+menuPosition: 2
+codeCopyButton: true
+aliases:
+  - "/docs/get-started/tree-quick-start/"
+  - "/start/tree-quick-start/"
+  - "/docs/tree-start/"
+
+---
+
+`SharedTree` is a [distributed data structure](../data-structures/overview.md) that looks and feels like simple JavaScript objects with a type safe wrapper.
```

Comment:
It makes me really sad that this document on DDSs doesn't even mention tree as an option. We should probably fix that before we link to it. I did let @nmsimons know, and he filed an item to update our docs to recommend tree as the default pattern.

Reply:
I could just take that item, seems like a simple enough change. I feel like it's also fine to just link to it anyways since it doesn't really change the meaning of this doc at all